README.md

```diff
@@ -14,4 +14,13 @@ This will only store technical metadata for files in the current version; techni
 Background processing is performed by [Sidekiq](https://github.com/mperham/sidekiq).
 
 Sidekiq can be monitored from [/queues](http://localhost:3000/queues).
-For more information on configuring and deploying Sidekiq, see this [doc](https://github.com/sul-dlss/DevOpsDocs/blob/master/projects/sul-requests/background_jobs.md).
\ No newline at end of file
+For more information on configuring and deploying Sidekiq, see this [doc](https://github.com/sul-dlss/DevOpsDocs/blob/master/projects/sul-requests/background_jobs.md).
+
+## Requirements
+### Siegfried
+[Siegfried](https://github.com/richardlehane/siegfried) is used for file identification.
+
+To install on OS X:
+```
+brew install mistydemeo/digipres/siegfried
```

Comment:
Is there a dockerized one we can invoke instead of having to install it locally?

Reply:
This would be tricky, since we're invoking siegfried from the commandline.

Reply:
Yeah, I should've read the whole PR before sharing that comment.